src/backend/impress/settings.py

```diff
@@ -619,8 +619,9 @@ def post_setup(cls):
                 release=get_release(),
                 integrations=[DjangoIntegration()],
             )
-            with sentry_sdk.configure_scope() as scope:
-                scope.set_extra("application", "backend")
+            # Add the application name to the Sentry scope
```

Comment:
```suggestion
            # Add "application" and "backend" tags the Sentry scope
```

Reply:
should it be : scope.set_tag("application", "backend")
application was opposed to "worker" for celery if I remember well

Reply:
>  I’ve since noticed that the API has changed to deprecate set_extra in favor of set_context,

Reply:
When using the same "project" in Sentry, it allows to tag frontend vs backend also. The tag application was used in "people" for this. 
I guess this is ok to be merged like this and improved later if needed?